src/plasma/plasma_manager.cc

```diff
@@ -43,6 +43,21 @@
 #include "state/task_table.h"
 #include "state/db_client_table.h"
 
+int handle_sigpipe(Status s, int fd) {
+  if (s.ok()) {
+    return 0;
+  }
+  if (errno == EPIPE || errno == EBADF || errno == ECONNRESET) {
+    ARROW_LOG(WARNING)
```

Comment:
Does it really make sense to convert the plasma manager to use arrow macros for logging? Are you planning on moving `plasma_manager.cc` into Arrow? Other than for checking status codes for the manager's interaction with the store?